src/main/java/persistence/sql/dml/query/clause/WhereClause.java

```diff
@@ -0,0 +1,24 @@
+package persistence.sql.dml.query.clause;
+
+import persistence.sql.dml.conditional.Criteria;
+
+import static persistence.sql.constant.SqlConstant.EMPTY;
+import static persistence.sql.constant.SqlFormat.WHERE;
+
+public class WhereClause {
+
+    private final Criteria criteria;
+
+    public WhereClause(final Criteria criteria) {
+        this.criteria = criteria;
+    }
+
+    public String toSql() {
+        String sql = criteria.toSql();
+
+        return EMPTY.getValue().equals(sql) ?
+                EMPTY.getValue() :
+                String.format(WHERE.getFormat(), criteria.toSql());
```

Comment:
```suggestion
        if(EMPTY.getValue().equals(sql)) {
             return EMPTY.getValue();
        }

        return String.format(WHERE.getFormat(), criteria.toSql());
```

삼항연산자도 if-else처럼 머릿속을 복잡하게 만드는 요소중에 하나라고 생각되어서
저는 빠른 리턴을 사용하는 편입니다 ^^

Reply:
if-else 해서 빠른 패턴 형태도 좋아하는데, 이런 형태가 지속되면 메소드를 private 하게 만든후 호출하는 형식으로 해서 따로만들어줘야하는 과정이 번거러워서 삼항연산자를 썼었는데 좋은 의견감사합니다.
다음 단계에서 수정 해보록 하겠습니다 😄